automation/vars/uni06zeta.yaml

```diff
@@ -0,0 +1,68 @@
+---
+vas:
+  uni06zeta:
+    stages:
+      - path: examples/dt/uni06zeta/control-plane/nncp
+        wait_conditions:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+            --timeout=60s
+
+          - >-
+            oc label node master-0 openstack.org/cinder-lvm=''
+
+          - >-
+            ssh hypervisor sudo ip link add link osp_trunk name vlan218 type vlan id 218
+
+          - >-
+            ssh hypervisor sudo ip addr add 172.38.0.1/24 dev vlan218
+
+          - >-
+            ssh hypervisor sudo ip link set dev vlan218 up
+
+          - >-
+            ssh hypervisor sudo iptables -A POSTROUTING -s 172.38.0.0/24 ! -d 172.38.0.0/24 -j MASQUERADE -t nat
```

Comment:
I think these need a post-run hook as well?

Reply:
There are few more options to evaulte